src/Compilers/Core/Portable/CorLightup.cs

```diff
@@ -231,6 +231,27 @@ internal static void RemoveAssemblyResolveHandler(Func<string, Assembly, Assembl
             {
                 GetOrRemoveAssemblyResolveHandler(handler, _AppDomain.remove_AssemblyResolve);
             }
+
+            private static class _Thread
+            {
+                internal static readonly Type Type = ReflectionUtilities.TryGetType("System.Threading.Thread");
+
+                internal static readonly MethodInfo Sleep = Type
+                    .GetTypeInfo()
+                    .GetDeclaredMethod("Sleep", typeof(TimeSpan));
+            }
+
+            private static Action<TimeSpan> s_sleepfn;
+
+            internal static void Sleep(TimeSpan timeSpan)
```

Comment:
@agocke @jaredpar @tmat this _is_ present on all the platforms we run on, right?